docs/release-notes.md

```diff
@@ -16,6 +16,7 @@ nav_order: 9
   flatcar 1.1.0-exp, openshift 4.13.0-exp)_
 - Allow specifying arbitrary LUKS open options _(fcos 1.5.0-exp,
   flatcar 1.1.0-exp, openshift 4.13.0-exp)_
+- Allow enabling Tang offline provisioning _(fcos 1.5.0-exp, openshift 4.13.0-exp)_
```

Comment:
The upstream release note was `Support offline Tang provisioning via pre-shared advertisement`.  We could reuse that here.

Reply:
Ah, yeah thats fair. Wasn't sure if we were being purposeful about 'Allow'. though might as well keep the old one.

Reply:
Not especially purposeful about it, no.